src/layout/focus_ring.rs

```diff
@@ -69,6 +70,8 @@ impl FocusRing {
 
         let color = if is_active {
             self.config.active_color
+        } else if is_urgent {
```

Comment:
Should this maybe take precedence over is_active? So that in the overview you can see urgent active windows on other workspaces with urgent color.